crates/witness-generator/src/eest_generator.rs

```diff
@@ -220,30 +222,137 @@ fn find_all_files_with_extension(path: &Path, extension: &str) -> Vec<PathBuf> {
 
 #[cfg(test)]
 mod tests {
+    use flate2::bufread::GzDecoder;
+    use tar::Archive;
+
     use super::*;
+    use std::{fs::File, str::FromStr};
 
-    #[test]
-    fn test_eest_generator_deletes_directory_on_drop() {
-        for delete_eest_folder in [false, true] {
-            let directory_path = PathBuf::from("./zkevm-fixtures/test-123");
-            std::fs::create_dir_all(&directory_path).unwrap();
-            assert!(directory_path.exists());
-
-            let eest_generator = ExecSpecTestBlocksAndWitnesses {
-                directory_path: directory_path.clone(),
-                include: vec![],
-                exclude: vec![],
-                delete_eest_folder,
-            };
+    fn decompress_eest_release(dest_dir: &Path) -> Result<()> {
+        let path =
+            PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("testdata/zkevm_fixtures_v0.1.0.tar.gz");
+        let file = File::open(path)?;
+        let buf_reader = std::io::BufReader::new(file);
+        let tar = GzDecoder::new(buf_reader);
+        let mut archive = Archive::new(tar);
+        archive.unpack(dest_dir)?;
+        Ok(())
+    }
 
-            drop(eest_generator);
+    fn prepare_downgraded_eest_fixtures(target_path: &Path) -> Result<()> {
```

Comment:
This helper is built on top of the previous function, which only leaves one single EEST fixture in the folder, making witness generation fast.

Whenever we have a release that includes the downgraded 1M gas cases, _maybe_ that will be easy enough to avoid this. I'll test as soon as that setup is doable.